packages/services/api/src/modules/audit-logs/module.graphql.ts

```diff
@@ -1,56 +1,22 @@
 import { gql } from 'graphql-modules';
 
 export const typeDefs = gql`
-  """
-  AuditLog is a record of actions performed by users in
-  the organization. It is used to track changes and
-  actions performed by users.
-  """
-  interface AuditLog {
-    id: ID!
-    eventTime: DateTime!
-    record: AuditLogIdRecord!
-  }
-
-  type AuditLogIdRecord {
-    userId: ID!
-    userEmail: String!
-    organizationId: ID!
-    """
-    User can be null if the user is deleted
-    """
-    user: User
-  }
-
   extend type Mutation {
     exportOrganizationAuditLog(
-      selector: OrganizationSelectorInput!
-      filter: AuditLogFilter!
+      input: ExportOrganizationAuditLogInput!
     ): ExportOrganizationAuditLogResult!
   }
 
+  input ExportOrganizationAuditLogInput {
+    selector: OrganizationSelectorInput!
+    filter: AuditLogFilter!
+  }
```

Comment:
We follow the convention of having a single input field named after the mutation field for Mutation fields.